lib/server.js

```diff
@@ -134,6 +137,104 @@ Server.prototype.start = function () {
     return this
   }
 
+  var self = this
+
+  const requiredFileExtensions = ['.dll', '.exe', '.so', '.txt']
+  const serverFolders = [
+    'addons',
+    'argo',
+    'battleye',
+    'contact',
+    'curator',
+    'dll',
+    'dta',
+    'enoch',
+    'expansion',
+    'gm',
+    'heli',
+    'jets',
+    'kart',
+    'mark',
+    'mpmissions',
+    'orange',
+    'tacops',
+    'tank'
+  ]
+  const symlinkFolders = serverFolders.concat(self.mods).concat(config.serverMods)
+
+  return fs.promises.mkdtemp(path.join(self.config.path, 'arma-server-'))
```

Comment:
Check it tmpdir can be used on Windows when server directory is on another drive